tests/drone/csp.yaml

```diff
@@ -3,7 +3,8 @@ directives:
     - '''self'''
   connect-src:
     - '''self'''
-    - 'https://${KEYCLOAK_DOMAIN:-keycloak:8443}/'
+    # In contrary to bash and docker the default is given after the | character
```

Comment:
```suggestion
    # In contrast to bash and docker the default is given after the | character
```

Reply:
I copied this from this PR in ocis https://github.com/owncloud/ocis/pull/9028. Should we keep it same or change it?